### ISSUE TITLE
Datasets

### ISSUE THREAD

**Issue by hsinweiyo** [denzelabad/OzFish-Classification#6]
It seems that the most of the images in OzFish dataset are offline. By any chance do you have a copy of the dataset?

**Comment by denzelabad** [comment]
Yes I do actually, the image folder is about 3.5gb total. I'm not sure how to send you a folder that large, maybe give me your email and I can upload them to a Google Drive folder.

**Comment by hsinweiyo** [comment]
> Yes I do actually, the image folder is about 3.5gb total. I'm not sure how to send you a folder that large, maybe give me your email and I can upload them to a Google Drive folder.

Sure, thank you so much!
My email is hsinweiyo@gmail.com

**Comment by denzelabad** [comment]
Okay I've uploaded the dataset to a Google Drive folder, it should be in your email. Let me know if you have trouble accessing it.
